NavigateU/NavigateU/HomeScreen/HomeDesignView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  HomeDesignView.swift
+//  NavigateU
+//
+//  Created by Faki Doosuur Doris on 17.03.2024.
+//
+
+import UIKit
+
+class HomeDesignView: UIView {
```

Comment:
one of best practice - use final keyword for classes, that will not have any subclasses. It's will be useful for code-reading and optimizations